job/validators.py

```diff
@@ -277,6 +278,12 @@ def cross_clean_pv(pvmodel):
                                                   self.models["Site"].__getattribute__("longitude")):
                         self.add_validation_error("Wind", "Site",
                               "latitude/longitude not in the WindToolkit database. Cannot retrieve wind resource data.")
+            
+            if self.models["Wind"].__getattribute__("operating_reserve_required_pct") == None:
```

Comment:
@rathod-b similar comment to above, we may also want to override user inputs if they provide operating reserve for wind and off_grid_flag is False.